src/components/DeleteAccountConfigConfirmationDialog/DeleteAccountConfigConfirmationDialog.jsx

```diff
@@ -171,7 +171,7 @@ const DeleteInstructions = () => {
             <StyledLink
               target="_blank"
               rel="noopener noreferrer"
-              href="https://youtu.be/UkOhNEJtkCU"
+              href="https://docs.omnistrate.com/getting-started/account-offboarding/?h=off"
```

Comment:
@Nithishprem no need to highlight "off" can we change the URL to  - https://docs.omnistrate.com/getting-started/account-offboarding/